src/SDKs/CognitiveServices/dataPlane/Vision/CustomVision/Training.Tests/Microsoft.Azure.CognitiveServices.Vision.CustomVision.Training.Tests.csproj

```diff
@@ -1,183 +1,47 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
   <PropertyGroup>
     <Description>Microsoft.Azure.CognitiveServices.Vision.CustomVision.Training.Tests Class Library</Description>
     <AssemblyName>Microsoft.Azure.CognitiveServices.Vision.CustomVision.Training.Tests</AssemblyName>
-    <Version>1.0.0</Version>    
+    <Version>1.0.0</Version>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TargetFrameworks>net452</TargetFrameworks>
```

Comment:
@areddish 
If you have specific needs for these project properties, could you please explain?
If this was done in mistake, please remove from line 10 to 22. We already have all the required project properties added as part of common project properties for the entire repository.
Please help us understand your needs or if this was in mistake, please remove these properties.

Reply:
Same as above, it was just so I could run tests and seemed necessary from the last time I ran them. I'll revert and investigate locally.